nebullvm/operations/conversions/converters.py

```diff
@@ -104,26 +106,33 @@ def pytorch_conversion(self):
 
 
 class ONNXConverter(Converter):
-    DEST_FRAMEWORKS = []
+    DEST_FRAMEWORKS = [DeepLearningFramework.NUMPY]
 
-    def execute(self, save_path, model_params):
-        onnx_path = save_path / f"{self.model_name}{self.ONNX_EXTENSION}"
-        try:
-            model_onnx = onnx.load(str(self.model))
-            onnx.save(model_onnx, str(onnx_path))
-        except Exception:
-            self.logger.error(
-                "The provided onnx model path is invalid. Please provide"
-                " a valid path to a model in order to use Nebullvm."
-            )
-            self.converted_models = []
-
-        self.converted_models = [str(onnx_path)]
+    def execute(
+        self,
+        save_path: Path,
+        model_params: ModelProto,
+    ):
+        self.converted_models = [self.model]
+        for framework in self.DEST_FRAMEWORKS:
+            if framework is DeepLearningFramework.NUMPY:
+                self.pytorch_conversion(save_path, model_params)
+            else:
+                raise NotImplementedError()
 
     def tensorflow_conversion(self):
         # TODO: Implement conversion from ONNX to Tensorflow
         raise NotImplementedError()
 
-    def pytorch_conversion(self):
-        # TODO: Implement conversion from ONNX to Pytorch
-        raise NotImplementedError()
+    def pytorch_conversion(self, save_path, model_params):
+        self.model_onnx = model_params
```

Comment:
This line is causing the error. It should be `self.model_onnx = self.model`, not `model_params`

Reply:
I used type of `ModelParams` instead of `ModelProto` in the convertors class.

Pushed a fix here - https://github.com/nebuly-ai/nebullvm/pull/168/commits/f44c2531a77fec36f59f7b79a280d2c01243a3aa

I think this works now

Reply:
It still doesn't work, btw `ModelParams` was the correct type of the `model_params` parameter. `model_params` is a class that contains info about the model, while self.model_onx should contain the path to the onnx model

Reply:
Ohh.. My Bad

Got it now, fixing it right away